packages/nimble-components/src/table-column/text-base/index.ts

```diff
@@ -36,6 +37,8 @@ export function mixinTextBase<
     TColumnValidator extends ColumnValidator<[]> = ColumnValidator<[]>
 >(base: TBase) {
     return mixinGroupableColumnAPI(
-        mixinFractionalWidthColumnAPI(mixinColumnWithPlaceholderAPI(base))
+        mixinFractionalWidthColumnAPI(
+            mixinColumnWithPlaceholderAPI(mixinSortableColumnAPI(base))
+        )
```

Comment:
nit
```suggestion
        mixinFractionalWidthColumnAPI(
            mixinColumnWithPlaceholderAPI(
                mixinSortableColumnAPI(base)
            )
        )
```

Reply:
Prettier does not agree with your suggestion 😄 

Assuming you don't feel really strongly about this specific formatting, I think I'll let prettier have its way here.

Reply:
Sometimes I hate prettier.